include/NovelRT/Exceptions/CharacterNotFoundException.h

```diff
@@ -0,0 +1,21 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_CHARACTER_NOT_FOUND_EXCEPTION_H
+#define NOVELRT_CHARACTER_NOT_FOUND_EXCEPTION_H
+
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
```

Comment:
Please remove these preprocessor errors from your exception types. I won't repeat this comment for all of them, but please do note this applies to all your exception types.